### DISCUSSION TITLE
Phase 2 Release

### DISCUSSION THREAD

**Discussion by 100hp4ever** [crypto-org-chain/chain-main#659] [Q&A] [answered]
Hi guys, I have a question regarding the phase 2 release. We had this proposal (https://crypto.org/explorer/proposal/5) a while ago which was rejected due to some issues found. I thought that this release is mandatory before the cronos go live. As far as I know each governance proposal takes 2 weeks to be completed. What is the plan now? Will it go live after the cronos launch or is a shorter voting duration planned?

**Comment by aw126** [comment, marked as answer]
Hello, thanks for your question and support. Yes, it was paused previously by unexpected tech issues that we found on cosmos SDK that provides un-deterministic behavior in the upgrade module https://github.com/cosmos/cosmos-sdk/pull/10189. We have not confirmed a specific date on this yet as the team is lately focusing on Cronos, but we will make an announcement to the community as soon as we get more clear about the plan on this.
